tools/mac-os-x-setup.source.sh

```diff
@@ -0,0 +1,11 @@
+# This script must be sourced. Please run `source tools/mac-os-x-setup.sh`
+
+# TODO: Remove sqlite once pkg_config is able to not bubble up a system path
+# For more information, see https://github.com/fxbox/foxbox/issues/414
+brew install openssl libupnp sqlite
+
+# openssl and sqlite libraries are already installed on OS X base system. That's
+# why we need to export them explicitely.
```

Comment:
nit: "that's why we need to explicitely specify the directories that brew installed"